src/components/Main.jsx

```diff
@@ -7,24 +7,21 @@ import NothingFound from './NothingFound';
 import KeyWordSearch from './KeyWordSearch';
 import NewsCardsList from './NewsCardsList';
 import { APIkey } from '../utils/constants';
-import { saveUserArticle } from '../mockData/SavedArticles';
-import { currentUserContext } from '../contexts/currentUserContext';
 
 const Main = ({ handleOpenLoginPopup, isLoggedIn }) => {
   const [loading, setLoading] = useState(false);
   const [articles, setArticles] = useState([]);
   const [error, setError] = useState(null);
   const [hasSearched, setHasSearched] = useState(false);
   const { setKeyword } = useContext(KeywordContext);
-  const { currentUser } = useContext(currentUserContext);
 
   const handleSearch = (searchKeyword) => {
     setLoading(true);
     setError(null);
     setHasSearched(true);
     setKeyword(searchKeyword);
 
-    fetch(`https://newsapi.org/v2/everything?q=${searchKeyword}&apiKey=${APIkey}`)
+    fetch(`https://nomoreparties.co/news/v2/everything?q=${searchKeyword}&apiKey=${APIkey}`)
```

Comment:
Any api requests should be done in `App`, and then the data should be passed into the props, according to the checklist